### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by JuliaSprenger**
## Motivation

There is no dedicated test requirement document specified, so the full requirements should include the requirements for test.
This PR also fixes a couple of typos on the way.

## Checklist

- [x] Have you checked our [Contributing](https://github.com/catalystneuro/nwb-conversion-tools/blob/master/docs/contribute.rst) document?
- [x] Have you ensured the PR description clearly describes the problem and solutions?
- [x] Have you checked to ensure that there aren't other open or previously closed [Pull Requests](https://github.com/catalystneuro/nwb-conversion-tools/pulls) for the same change?
- [x] If this PR fixes an issue, is the first line of the PR description `fix #XXX` where `XXX` is the issue number?

# Update
The requirement issue is already fixed in `main`, this PR only adds typo fixes

### CODE REVIEW

File: requirements-full.txt [hunk 1]
```diff
@@ -25,3 +25,5 @@ ndx-dandi-icephys>=0.4.0
 pillow==9.1.1
 pyopenephys==1.1.2
 scanimage-tiff-reader==1.4.1
+parameterized
+pytest
```
[the remark this answers is not in the document]
Comment (CodyCBakerPhD) [reply requirements-full.txt:29]: I'll try to get #554 merged today so we can update this branch, remove the changes to `requirements-full.txt`, and then merge these typo fixes.

Thanks!

**Comment by CodyCBakerPhD** [comment]
It's so odd that the CI isn't finding the DANDI_API_KEY secret: https://github.com/catalystneuro/nwb-conversion-tools/runs/7342649698?check_suite_focus=true

I suppose it's because this is submitted from @JuliaSprenger's fork, but the actions appear to be going through the main repo...

File: requirements-full.txt (hunk 1 above)
Comment (CodyCBakerPhD) [inline requirements-full.txt:30]:
```suggestion
```

**Comment by CodyCBakerPhD** [comment]
Wow, hmm... OK I had figured that it was possible to skip the entire test class that way, but apparently not...

@JuliaSprenger So sorry, but can you try skipping just that one function instead https://github.com/catalystneuro/nwb-conversion-tools/blob/main/tests/test_internals/test_tools.py#L209?

**Comment by JuliaSprenger** [comment]
Hi @CodyCBakerPhD Now only the issue of secret tokens not being available in PR triggered runs remains. It's a security feature to prevent secrets to be accessible by non-project administrators. A safe way on how to still have the key available is to use the `pull_request_target` trigger in combination with a manual label of the PR by an administrator, e.g. like I did it here: https://github.com/INT-NIT/DigLabTools/blob/main/.github/workflows/run_tests.yml

**Comment by CodyCBakerPhD** [comment]
> A safe way on how to still have the key available is to use the pull_request_target trigger in combination with a manual label of the PR by an administrator

Looks like that is indeed working! Thanks!

**Pull request merged**